send/generic_send

```diff
@@ -59,6 +59,16 @@ case $DESTINATION_TYPE in
 		HOSTNAME=`echo $HOST | sed -e 's/^.*@//'`
 		PORT=`echo $DESTINATION | sed -e 's/^.*://'`
 		;;
+	url)
+		HOSTNAME="$DESTINATION"
+		HOST="$DESTINATION"
+		TRANSPORT_COMMAND="curl -i -H Content-Type:application/tar -f -X PUT --data-binary - "
```

Comment:
If we override TRANSPORT_COMMAND for service on line 18 (by including conf/sh file), it gets overwritten again by this in case of URL destination type. This definition for URL destination type should be above line 18. Somebody might want to pass certificates or login/pass for this connection and I think it's part of curl command.